crates/holochain_types/src/record.rs

```diff
@@ -445,6 +446,51 @@ impl WireRecord {
     }
 }
 
+/// Helper trait for the [`RecordBuilder`].
+pub trait RecordBuilderHelper {
+    /// Build this element into it's parts for constructing an [`Element`].
+    fn build(self) -> (EntryDefLocation, EntryVisibility, Entry);
+}
+
+impl RecordBuilderHelper for RecordBuilder {
+    fn build(self) -> (EntryDefLocation, EntryVisibility, Entry) {
```

Comment:
Does this really need to be a trait? Could it just be a method directly on `RecordBuilder` (or whatever we call it)?

Also for the same reason I'd like this not to be called a Builder, I'd like this method to not be called `build`.

```suggestion
impl RecordBuilderHelper for RecordBuilder {
    fn into_parts(self) -> (EntryDefLocation, EntryVisibility, Entry) {
```